metricflow/dataflow/builder/dataflow_plan_builder.py

```diff
@@ -100,6 +106,10 @@
 
 logger = logging.getLogger(__name__)
 
+_MEASURE_RECIPE_CACHE_SIZE = 1000
```

Comment:
Curious if we've put much thought into what the size should be? I guess we'll just experiment and tune it accordingly?

Reply:
These are based on rough sizes of semantic manifests and performance tests on the larger ones we've seen. We do need to encapsulate all these caches into a single object and add instrumentation.